ping-protect/src/androidTest/java/org/forgerock/android/auth/PingOneProtectEvaluateCallbackTest.kt

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright (c) 2024 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+package org.forgerock.android.auth
+
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import org.forgerock.android.auth.callback.TextOutputCallback
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Assert
+import org.junit.FixMethodOrder
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.MethodSorters
+
+@RunWith(AndroidJUnit4::class)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+class PingOneProtectEvaluateCallbackTest : BasePingOneProtectTest() {
+    private val TREE = "TEST_PING_ONE_PROTECT_EVALUATE"
```

Comment:
private val tree = "TEST_PING_ONE_PROTECT_EVALUATE"

property variable names are lowerCamelCase
const are UPPERCASE

Reply:
Refactored this to:
`private val authenticationTree = "TEST_PING_ONE_PROTECT_EVALUATE"`